appium/tests/specs/mock/setup/CheckRefreshKeyFromEkmWithPassPhrasePrompt.spec.ts

```diff
@@ -33,13 +33,16 @@ describe('SETUP: ', () => {
       };
       await AppiumHelper.restartApp(processArgs);
       await RefreshKeyScreen.waitForScreen(true);
-      await RefreshKeyScreen.fillPassPhrase('wrong passphrase');
-      await RefreshKeyScreen.clickOkButton();
-      await BaseScreen.checkModalMessage(CommonData.refreshingKeysFromEkm.wrongPassPhrase);
-      await RefreshKeyScreen.clickSystemOkButton();
-      await RefreshKeyScreen.cancelRefresh();
-      await KeysScreen.openScreenFromSideMenu();
-      await KeysScreen.checkKeysScreen([ekmKeySamples.key0]);
+      // TODO: Temporarily disable the logic below as the cancel button on the refresh screen
+      // is not functioning correctly in Semaphore CI for some reason.
+      // https://flowcrypt.semaphoreci.com/workflows/7310b5b7-d544-428c-82ce-7f7f060e7e60?pipeline_id=186f2ad1-5d22-44d3-ba07-76deaf3ea49f
```

Comment:
@sosnovsky Have any idea why this test fails (can't click button even though elements are visible on the screen)

Reply:
Can be caused by some changes in iOS behavior which were adapted in the newer Appium version.
After upgrading to the latest Appium we can re-check if it works there.